Client.xcodeproj/project.pbxproj

```diff
@@ -5920,20 +5922,22 @@
 		F84B21D61A090F8100AAB793 /* ClientTests */ = {
 			isa = PBXGroup;
 			children = (
-				43446CF12412F9A800F5C643 /* ETPCoverSheet */,
```

Comment:
Did you sort this folder in XCode? Usually that's not a bad idea but I suggest to revert this to not run into future merge conflicts of the .pbxproj file

Reply:
ok